### ISSUE TITLE
Incompatible licenses

### ISSUE THREAD

**Issue by aehlke** [jeffwidman/ansible-tarsnap#1]
> The tasks for installing Tarsnap originally pulled from:
> https://github.com/sovereign/sovereign/blob/master/roles/tarsnap/tasks/tarsnap.yml

FYI sovereign is GPLv3, your library is MIT :(

**Comment by jeffwidman** [comment]
Indeed, this project should be GPLv3. I hadn't noticed that originally as IIRC parts of this were assembled from several other playbooks around the internet (I wasn't happy with any of the existing ones) and so I never looked at the `sovereign` source. 

Would you be willing to submit a PR fixing it?

**Comment by aehlke** [comment]
~~If you'd like to keep your parts as MIT, perhaps we could refactor the GPLv3 bits to be an external dependency instead?~~ nevermind, not sure that makes any sense...
